content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1067 @@
+{
```

Comment:
<p>Item 4 is missing now :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
Got it 😃 1, 2, 3, 4 (not 5)